tests/test_printers.py

```diff
@@ -15,6 +15,7 @@
 import pytest
 
 from substra.cli import printers
+from substra.cli.interface import GlobalConf
```

Comment:
I don't think these changes are needed anymore. Your last refactor removed changes from `printers.py` so the associated unit tests shouldn't change.

Reply:
You're right, I'll remove this! 🙂